src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
This isn't really user facing though, and it seems to have added a lot of complexity to the ``addConstraint`` macro implementation. Do you also need JuMPDicts of constraint objects?